modules/apps/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/view_page_details.jsp

```diff
@@ -199,12 +197,16 @@ PortalUtil.addPortletBreadcrumbEntry(request, LanguageUtil.get(request, "details
 											<portlet:param name="title" value="<%= String.valueOf(wikiPage.getTitle()) %>" />
 										</portlet:actionURL>
 
-										<liferay-ui:icon
+										<clay:button
+											aria-label='<%= LanguageUtil.get(request, "unsubscribe") %>'
+											borderless="<%= true %>"
+											data-url="<%= unsubscribeURL %>"
```

Comment:
Instead of setting data-x attributes, the pattern we should follow is to add an `additionalProps` map. 
```
<clay:button
	additionalProps='<%=
		HashMapBuilder.<String, Object>put(
		    "url", unsuscribeURL
		).build() 
        %>'
       ....
/>
```

So then your propsTransformer would be like: 

```
export default function propsTransformer({additionalProps, ...props}) {
	return {
		...otherProps,
		onClick: () => {
			submitForm(document.href, additionalProps.url);
		},
	};
}
```

Anyway, is the `submitForm` really needed? Couldn't we change this button to be a `<clay:link>` with `icon` and set the `href` attribute to the `unsubscribeURL`? I think this makes more sense, also because that is the way it is currently workin with `<liferay-ui:icon>`, so we can get rid of the propsTransformer. Same would apply to the susbcribe button.

Reply:
It is because in IconTag.java, we are doing:

```
if (isForcePost()) {
			onClick = StringBundler.concat(
				"event.preventDefault();", onClick,
				"submitForm(document.hrefFm, '", HtmlUtil.escapeJS(getUrl()),
				"')");
		}
```
So, it should be a button since it is executing an action, clay:link are for navigation, right?